### PR TITLE
Redshift server less stack

### PR DESCRIPTION
**Pull request by bwighane**
- The hard coded values will be removed, I left them should you want to deploy the stack in the dev environment

### CODE REVIEW

**Comment by bwighane** [comment]
I will be removing the bin  folder for the virtual environment very shortly as you review

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-aws-cdk.aws-logs~=1.109.0
-aws-cdk.aws-s3~=1.109.0
-aws-cdk.aws-s3-assets~=1.109.0
-aws-cdk.aws-s3-deployment~=1.109.0
-aws-cdk.aws-s3-notifications~=1.109.0
-aws-cdk.aws-secretsmanager~=1.109.0
-aws-cdk.aws-ssm~=1.109.0
-aws-cdk.core~=1.109.0
-aws-cdk.pipelines~=1.109.0
+aws-cdk.aws-cloudformation~=1.181.1
```
Comment (BranfordTGbieor) [inline requirements.txt:1]: @bwighane I am just going to merge and see but I suspect the adjustment of these versions might break something.

File: app.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
         'region': deployment_region,
     }
     logical_id_prefix = get_logical_id_prefix()
-    
+
```
Comment (BranfordTGbieor) [inline app.py:31]: @bwighane why the extra space here?

File: requirements.txt (hunk 1 above)
Comment (bwighane) [reply requirements.txt:1]: I understand. 'cdk synth' worked after making corrections to the requirements. Let me know how that goes.

File: app.py (hunk 2 above)
Comment (bwighane) [reply app.py:31]: It should have been added by a file formatter and it makes sense. I can remove it as well easily.

File: requirements.txt (hunk 1 above)
Comment (BranfordTGbieor) [reply requirements.txt:1]: sure!

**Review by BranfordTGbieor — APPROVED** [review]
Aside from some reservations about package versioning, looks good to merge.

**Pull request merged**